magda-ckan-connector/src/index.ts

```diff
@@ -22,6 +22,11 @@ const argv = addJwtSecretFromEnvVar(
             type: "string",
             demandOption: true
         })
+        .option("retrievedAt", {
```

Comment:
Harvesting can take a long time.  Ideally the `retrievedAt` date would indicate when the request for that particular dataset was initiated, not when the overall process began.